### ISSUE TITLE
Java code to parse this metadata

### ISSUE THREAD

**Issue by shevek** [nebula-plugins/gradle-info-plugin#21]
Hi,

I have written Java code to run within a deployed application and parse metadata published by the gradle-info-plugin and build a support bundle. So:

Is there standard code available already for this task?

Would you like me to contribute my code to a Nebula repository, or just publish it on my own?

Thank you.

**Comment by nadavc** [comment]
Hi Shevek. This sounds like something that would be appropriate for application code instead of plugin code. Where would you suggest we put this code?

**Comment by rspieldenner** [comment]
Options:

1. We set up an info client project under the nebula-plugins org
2. We make this a multiproject and have a plugin and client subproject
3. somewhere else

**Comment by shevek** [comment]
Yes, it's definitely application code, not plugin code. Right now it's built into one of my applications, and I want to split it out and have it available to all my applications. I think @rspieldenner has enumerated the options perfectly.

If you host it here, would I be able to continue as a contributor, as I have commercial requirements for the accuracy of the code. Alternatively, I am happy to publish to Central under org.anarres.* and keep the code maintained.

**Comment by shevek** [comment]
Code now published here, as it seemed to make sense to put it into a diagnostic support bundle builder.

https://github.com/shevek/jdiagnostics/blob/master/src/main/java/org/anarres/jdiagnostics/ProductMetadata.java

Will be on Central soon.
